.ibm/pipelines/resources/rhdh-operator/rhdh-start-rbac_K8s.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Backstage
+apiVersion: rhdh.redhat.com/v1alpha3
+metadata:
+  name: rhdh-rbac
+spec:
+  application:
+    imagePullSecrets:
+      - "rh-pull-secret"
+    route:
+      enabled: false
+    image: 'quay.io/rhdh/rhdh-hub-rhel9:next'
```

Comment:
@zdrapela quay.io/rhdh/rhdh-hub-rhel9:next should be changed to  quay.io/rhdh-community/rhdh:next

Reply:
Thank you. I missed this during the rebases